### DISCUSSION TITLE
How to resume the job?

### DISCUSSION THREAD

**Discussion by GTAPeople** [illsk1lls/ZipRipper#32] [Q&A]
I have an encrypted Zip file which took long days but unfortunately it was aborted.
Then I started again then it popped to ask to resume the job.

I clicked YES then it closed without resume the old data. I had to record a video because the window closed too fast after it.
I have been trying because I don't want to clear the data then start over again for days.

It says "system cannot find the file pwhash" despite I have pwhash on this folder "C:\Users\MyUsername\AppData\Roaming\ZR-InProgress"

How I can fix it?

**Comment by illsk1lls** [comment]
Are there any special chars in your username?

**Reply by GTAPeople** [reply]
No, it's just "gtape"

**Comment by illsk1lls** [comment]
I just did a push that may fix the issue

**Reply by GTAPeople** [reply]
It didn't work, even I cleared the resume data

**Reply by illsk1lls** [reply]
I am unsure what is going on i need to add logging for issues like this.. I'll do my best to get it in asap
